src/controllers/update/remove.cannot.use.controller.ts

```diff
@@ -4,7 +4,7 @@ import { logger } from "../../utils/logger";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
-    logger.debugRequest(req, `GET ${config.SIGN_OUT_PAGE}`);
+    logger.debugRequest(req, `GET ${config.REMOVE_CANNOT_USE_PAGE}`);
```

Comment:
This change looks like nothing to do with 'save and resume'. Is it correct? (I imagine this question cropped up previously on PR to feature branch....)

Reply:
I think that was something I came across when debugging the tests, i noticed the page name didn't match the controller (must have been copied and pasted). I can change it back if you want.

Reply:
No, all good.